test/import/cdk8s-template.yaml

```diff
@@ -0,0 +1,4 @@
+language: typescript
```

Comment:
Prefer to use create this file in the test itself - if its simple enough. That is, write this file to disk from within the test.